core/common/config/config.go

```diff
@@ -259,6 +259,7 @@ type NodeConfig struct {
 type KernelConfig struct {
 	MinNewChainAmount string          `yaml:"minNewChainAmount,omitempty"`
 	NewChainWhiteList map[string]bool `yaml:"newChainWhiteList,omitempty"`
+	IsCreateChain     bool            `yaml:"isCreateChain,omitempty"`
```

Comment:
`enableCreateChainWhiteList`  maybe a more clear name. or restructure `NewChainWhiteList` to following?

``` go
type KernelConfig struct {
  ....
  NewChainWhiteList struct {
      Enable bool
      Addrs map[string]bool
  }
  ....
}
```

Reply:
Thanks, useing `enableCreateChainWhiteList ` here.  Not use `NewChainWhiteList ` struct for backward compatibility.